docs/src/user_docs/stdlib/crypto/fri.md

```diff
@@ -0,0 +1,10 @@
+# FRI verification procedures
+Namespace `std::crypto::fri` contains modules for verifying [FRI](https://eccc.weizmann.ac.il/report/2017/134/) proofs.
+
+## FRI Extension 2, Fold 4
+
+Module `std::crypto::fri::frie2f4` contains procedures for verifying FRI proofs generated over quadratic extension of the Miden VM's base field. Moreover, the procedures assume that layer folding during commit phase of FRI protocol was performed using folding factor 4.
```

Comment:
nits: missing words - "...generated over the quadratic extension of..." and "...during the commit phase of the FRI protocol..."